zerver/forms.py

```diff
@@ -65,12 +66,12 @@ def email_is_not_mit_mailing_list(email: str) -> None:
             else:
                 raise AssertionError("Unexpected DNS error")
 
-def check_subdomain_available(subdomain: str, from_management_command: bool=False) -> None:
+def check_subdomain_available(subdomain: str, allow_reserved_subdomain: bool=False) -> None:
     error_strings = {
         'too short': _("Subdomain needs to have length 3 or greater."),
         'extremal dash': _("Subdomain cannot start or end with a '-'."),
         'bad character': _("Subdomain can only have lowercase letters, numbers, and '-'s."),
-        'unavailable': _("Subdomain unavailable. Please choose a different one.")}
+        'use flag': _("pass --allow-reserved-subdomain as flag to use reserved names")}
```

Comment:
Let's call this one `reserved subdomain`.

Reply:
Ok